signup.html

```diff
@@ -11,42 +11,56 @@
     />
     <link rel="stylesheet" href="styles/account/account.css" />
     <title>판다마켓 - 회원가입</title>
-    <link rel="icon" href="/images/icon_favicon.ico" />
+    <link rel="icon" href="/images/icon_favicon.png" />
   </head>
 
   <body>
-    <div class="loginContainer">
+    <div class="login-container">
       <header>
         <a href="/">
-          <img
-            src="images/img_logo.png"
-            alt="판다마켓 로고"
-            class="image-logo"
-          />
+          <img src="images/logo.png" alt="판다마켓" class="image-logo" />
         </a>
       </header>
 
-      <form>
-        <label>이메일 <input type="email" /></label>
-        <label>닉네임 <input type="text" /></label>
-        <label
-          >비밀번호 <input type="password" />
-          <img
-            src="/images/btn_visibility_on.png"
-            alt="비밀번호 보기"
-            class="passwordVisibilityOn"
+      <form class="form-account">
+        <label class="label-account">
+          이메일
+          <input
+            type="email"
+            placeholder="이메일을 입력해주세요"
+            class="input-account"
+          />
+        </label>
+        <label class="label-account">
+          닉네임
+          <input
+            type="text"
+            placeholder="닉네임을 입력해주세요"
+            class="input-account"
+          />
+        </label>
+        <label class="label-account">
+          비밀번호
+          <input
+            type="password"
+            placeholder="비밀번호를 입력해주세요"
+            class="input-account password"
           />
+          <button class="btn-visibility-on"></button>
```

Comment:
❗️ 수정요청
해당 버튼에 type이 명시되어 있지않아, default type인 submit 버튼이 됩니다~
그래서 버튼을 클릭하면 form이 제출되네요! type을 아래처럼 명시해주세요!

```suggestion
          <button class="btn-visibility-on" type="button"></button>
```